README.md

```diff
@@ -1061,6 +1061,8 @@ Utilities for managing game servers.
 
 **[`^        back to top        ^`](#)**
 
+A [gateway](https://en.wikipedia.org/wiki/Gateway_(telecommunications)) is a piece of networking hardware or software used in telecommunications networks that allows data to flow from one discrete network to another.
```

Comment:
Software listed here does not fit this description - the section is not about network gateways, but a mish-mash of remote desktop servers, protocol "adapters" (e.g. SSH over HTTP), jumpboxes/bastions, VPNs, in-browser operating systems... I mentioned this in a previous PR.

I don't know what should be done exactly:
- rename the section? It will be hard since it's so mixed up.
- split it into more meaningful sections/categories? We would have to find which ones exactly (`Terminal Sharing` is an obvious one, maybe `Remote Access`, don't know about the others). Also _Any category you are creating has the minimum requirement of 3 items._
- move all items to `Miscellaneous`? Or just items which do not clearly fit in a category?
- leave it as-is (no description) and wait for https://github.com/awesome-selfhosted/awesome-selfhosted/milestone/2 to be completed which will allow more fine-grained/multiple categories for each item.

Last 2 solutions look OK to me.

Reply:
> Software listed here does not fit this description - the section is not about network gateways, but a mish-mash of remote desktop servers, protocol "adapters" (e.g. SSH over HTTP), jumpboxes/bastions, VPNs, in-browser operating systems... I mentioned this in a previous PR.
> 
> I don't know what should be done exactly:
> 
> * rename the section? It will be hard since it's so mixed up.
> * split it into more meaningful sections/categories? We would have to find which ones exactly (`Terminal Sharing` is an obvious one, maybe `Remote Access`, don't know about the others). Also _Any category you are creating has the minimum requirement of 3 items._
> * move all items to `Miscellaneous`? Or just items which do not clearly fit in a category?
> * leave it as-is (no description) and wait for https://github.com/awesome-selfhosted/awesome-selfhosted/milestone/2 to be completed which will allow more fine-grained/multiple categories for each item.
> 
> Last 2 solutions look OK to me.

I'd say merge without a description and over the next couple of days ill clean it up and push those which dont fit in a category into `Miscellaneous` while also cleaning up there if I can, there's a couple which have commit dates >2021 anyway

Reply:
:+1: